Customizing/global/plugins/Services/Repository/RepositoryObject/ReportWBDPoints/classes/class.ilObjReportWBDPointsGUI.php

```diff
@@ -9,6 +9,8 @@
 * @ilCtrl_Calls ilObjReportWBDPointsGUI: ilCommonActionDispatcherGUI
 */
 class ilObjReportWBDPointsGUI extends ilObjReportBaseGUI {
+	const CMD_SHOW_CONTENT = "showContent";
```

Comment:
Hat das hier irgendeinen tiefen Grund? Wird diese Konstante ausserhalb dieser Klasse verwendet? Ich vermute ja, dass das eher in die Base Klasse gehört...kann also später dort eingefactort werden und ist hier vorerst nur Ballast...